frontend/src/pages/loginPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { createRef, FormEvent, } from "react";
+import axios from "axios";
+import { InputText } from "primereact/inputtext";
+import { Button } from "primereact/button"
+import { Messages } from 'primereact/messages';
+
+interface LoginState {
+  password: string;
+}
+
+class LoginPage extends React.Component<{}, LoginState> {
+  private wrongPasswordRef: React.RefObject<Messages>
+  constructor(props = {}) {
+    super(props);
+    this.state = { password: ""};
+    this.wrongPasswordRef = createRef<Messages>()
+
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onChange = (event: FormEvent<HTMLInputElement>): void => {
+    this.setState({ password: event.currentTarget.value });
+  };
+
+  // Hits the token endpoint, and stores the token in local storage. Displays incorrect password text if error returned from endpoint
+  onSubmit = (event: FormEvent<HTMLFormElement>): void => {
+    const reqOpts = {
+      url: process.env.REACT_APP_API_ENDPOINT_TOKEN,
+      headers: { "Content-Type": "application/json" },
+      method: "POST",
+      data: JSON.stringify({
+        email: "hk196@duke.edu",
+        password: this.state.password,
```

Comment:
Were you able to test for the password to be accepted or recieve a affirmative response

Reply:
Everything else for frontend looks good. I was able to run it, can't say anything for authentication itself without the backend people.

Reply:
Yes, I checked by logging `response.data.access` (the token) to the console. I also checked that an incorrect password led to no token and an error message